lms/templates/footer.html

```diff
@@ -103,9 +103,9 @@
 
         <div class="wrapper-logo">
             <div class="navbar-brand">
-                <a href="https://web.mit.edu/" rel="noopener noreferrer" target="_blank" class="mit-link"></a>
-                <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link"></a>
-                <img  class="logo" src="${footer['logo_image']}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+                <a href="${settings.MARKETING_SITE_BASE_URL}">
+                  <img class="logo mitxonline-ol-logo" src="${footer['logo_image']}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
```

Comment:
I think the CSS size isn't getting picked here, I'm seeing an image of 164x42.
<img width="388" alt="image" src="https://github.com/mitodl/mitxonline-theme/assets/34372316/5e25c712-f0f1-4632-8b8e-ef59bd57544b">